src/components/footer.jsx

```diff
@@ -0,0 +1,55 @@
+import { React } from "react";
+
+function Footer() {
+    return (
+        <footer class="bg-white">
+            <hr />
+            <div class="w-full mx-auto container md:p-6 p-4 md:flex md:items-center md:justify-between">
+                <div className="sm:items-center">
+                    <div className="flex mt-4 space-x-6 sm:justify-center sm:mt-0">
+                        <a href="#" className="text-gray-500 hover:text-gray-900 dark:hover:text-white">
+                            <svg className="w-5 h-5" fill="currentColor" viewBox="0 0 24 24" aria-hidden="true"><path fill-rule="evenodd" d="M22 12c0-5.523-4.477-10-10-10S2 6.477 2 12c0 4.991 3.657 9.128 8.438 9.878v-6.987h-2.54V12h2.54V9.797c0-2.506 1.492-3.89 3.777-3.89 1.094 0 2.238.195 2.238.195v2.46h-1.26c-1.243 0-1.63.771-1.63 1.562V12h2.773l-.443 2.89h-2.33v6.988C18.343 21.128 22 16.991 22 12z" clip-rule="evenodd" /></svg>
+                            <span className="sr-only">Facebook page</span>
```

Comment:
please, import icon name, not pasting svg format.
This way will hanlde kebab-case words to camelCase, whitch is suitable for jsx format.